pom.xml

```diff
@@ -4,13 +4,24 @@
 
     <groupId>io.rtr</groupId>
     <artifactId>conduit</artifactId>
-    <version>1.14-SNAPSHOT</version>
+    <version>1.16-SNAPSHOT</version>
     <name>Conduit</name>
     <description>
         A simple abstraction over the rabbitmq java client
     </description>
     <url>www.renttherunway.com</url>
 
+    <developers>
+        <developer>
+            <name>Gene Trog</name>
```

Comment:
Return of the king 😛

Reply:
Why only me!? And I don't know this `Gene Trog` person ;)